src/ui/Base/Initialization.ts

```diff
@@ -645,8 +646,11 @@ export class LazyInitialization {
     }
   }
 
-  public static buildErrorCallback(chunkName: string) {
-    return () => LazyInitialization.logger.error(`Cannot load chunk for ${chunkName}. You may need to configure the paths of the ressources using Coveo.configureRessourceRoot. Current path is ${__webpack_public_path__}.`);
+  public static buildErrorCallback(chunkName: string, resolve: Function) {
+    return () => {
```

Comment:
Have you considered using `(error: Error) =>` to get the error from `require.ensure`? It could be useful information.